src/universal/RSE-RTC.c

```diff
@@ -0,0 +1,135 @@
+#include <pksm.h>
```

Comment:
As I posted on Discord:
> I have not yet added `GBA-RTC.c` to the next release of PKSM because I would like more feedback on whether it works as-is and if/how to improve it.

Reply:
Well, it's in its current version a year and a half old, and I think there hasn't been any feedback. You will have a greater chance of receiving any upon inclusion.